src/components/UserSignIn.jsx

```diff
@@ -0,0 +1,42 @@
+import { signInWithEmailAndPassword } from 'firebase/auth';
+import React, { useState } from 'react';
```

Comment:
it is not necessary to import React

Reply:
need to setEmail and setPassword using useState

Reply:
I meant to change `import  { useState } from 'react';` but it's fine its not a big deal